src/Components/Atbash/AtbashInput.js

```diff
@@ -19,45 +42,115 @@ class Atbash extends Component {
     return letters;
   };
 
+  putCharsInSpans = (string) => {
+    let map = Array.prototype.map;
+    let spans = map.call(string, char => {
+        if (char === " ") {
+            return <span>&nbsp;</span>
+        } else {
+            return <span>{char}</span>;
+        }
+    });
+    return spans;
+  }
```

Comment:
Just a note that putCharsInSpans does a little bit more than call map on the string array.

It actually uses Array.prototype.map, and then calls .call() on it, to create the nice gradual effect Lisha animated. 

To fit in one line it will look like

`{Array.prototype.map.call(ciphertext, c => (c === " ") ? <span>&nbsp;</span> : <span key={c}>{c}</span>)}`

Which works too, but I'm not sure if that's as readable than the helper function.